### PR TITLE
Dev experience master

### PR DESCRIPTION
**Pull request by mgencur**

### CODE REVIEW

File: openshift/e2e-common.sh [hunk 1]
```diff
@@ -144,75 +170,25 @@ function install_knative_eventing(){
 
 function run_e2e_tests(){
   header "Running tests with Multi Tenant Channel Based Broker"
+
+  local test_name=$1
+  local failed=0
+  local channels=messaging.knative.dev/v1alpha1:InMemoryChannel,messaging.knative.dev/v1alpha1:Channel,messaging.knative.dev/v1beta1:InMemoryChannel
```
Comment (matzew) [inline openshift/e2e-common.sh:176]: this is wrong, see: 
https://github.com/openshift/knative-eventing/pull/646/files#diff-cd758740b9c6da1c23340da7084242feL150

```
-channels=messaging.knative.dev/v1beta1:Channel,messaging.knative.dev/v1beta1:InMemoryChannel
```

The `v1alpha1` has been removed

**Comment by matzew** [comment]
Apply the change, and I think this is OK.

However, I'd like to test it against `release-next`, to get some results ;-) 

That said, `release-next` needs updates on `openshift/release`.

I leave it to @lberk to approve this

File: openshift/e2e-common.sh (hunk 1 above)
Comment (mgencur) [reply openshift/e2e-common.sh]: Updated. Thanks.

**Comment by lberk** [comment]
@mgencur any way we could get a test of this on release-next?  I think it makes sense but just want to make sure ci still passes.

As an aside, as somebody that always forgets this kind of thing exists, it'd be great to have some docs on how to source/run the tests when I come back later to run it.

**Comment by mgencur** [comment]
@lberk sure, I'll send this as a test PR against release-next. Regarding docs. Isn't it sufficient what is written in the Makefile?

**Comment by mgencur** [comment]
PR against release-next: https://github.com/openshift/knative-eventing/pull/650

**Comment by mgencur** [comment]
Some tests in https://github.com/openshift/knative-eventing/pull/650 failed but I'd say it's not caused by my changes.

**Comment by mgencur** [comment]
Tests in https://github.com/openshift/knative-eventing/pull/650 passed! Let's merge this. It already has all commits from 650

**Pull request merged**